### PR TITLE
Fix/m2 1055 fix number bullet options in wysiwyg

### PR DESCRIPTION
**Pull request by alex-i-e**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -38,8 +38,23 @@
...
+    "markdown-it-images-preview": "^1.0.1",
+    "markdown-it-imsize": "^2.0.1",
+    "markdown-it-ins": "^3.0.1",
+    "markdown-it-katex-external": "^1.0.0",
+    "markdown-it-mark": "^3.0.1",
+    "markdown-it-sub": "^1.0.0",
+    "markdown-it-sup": "^1.0.0",
+    "markdown-it-task-lists": "^2.1.1",
     "mixpanel-browser": "^2.47.0",
+    "path-browserify": "^1.0.1",
```
Comment (muhurov) [inline package.json:57]: do you really need this?

**Review by muhurov — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (alex-i-e) [reply package.json]: exactly, I 've removed redundant one

File: src/shared/components/MarkDownEditor/MarkDownEditor.utils.ts [hunk 2]
```diff
@@ -1,38 +1,27 @@
-import MdEditor from 'md-editor-rt';
+import { config } from 'md-editor-rt';
+
```
Comment (IhorNesterenko99) [inline src/shared/components/MarkDownEditor/MarkDownEditor.utils.ts:2]: pls remove spacing

File: src/shared/components/MarkDownEditor/MarkDownEditor.utils.ts [hunk 3]
```diff
@@ -1,38 +1,27 @@
...
-  VIDEO_LINK_REGEX,
-  VideoSourcePlayerLinks,
-  VideoSources,
-  YOUTUBE_VIDEO_ID_REGEX,
-} from './MarkDownEditor.const';
+import { LANGUAGE_BY_DEFAULT } from './MarkDownEditor.const';
 
 const { t } = i18n;
 
-export const getValidMediaFormatsRegex = (validFileTypesArray: string[]) => {
```
Comment (IhorNesterenko99) [inline src/shared/components/MarkDownEditor/MarkDownEditor.utils.ts:17]: Does .vebm, .mp4, youtube, vimeo video have correct display in the right column after changes?

File: src/shared/components/MarkDownEditor/MarkDownEditor.utils.ts (hunk 2 above)
Comment (alex-i-e) [reply src/shared/components/MarkDownEditor/MarkDownEditor.utils.ts]: done

File: src/shared/components/MarkDownEditor/MarkDownEditor.utils.ts [hunk 4]
```diff
@@ -78,177 +66,53 @@ MdEditor.config({
...
-
-          return videoId ? getVideoIframe(videoId, VideoSources.Vimeo, text) : defaultReturn;
-        }
+  markdownItConfig(mdit) {
+    mdit
+      .set({
+        html: true, // Enable HTML tags in source
+        xhtmlOut: true, // Use '/' to close single tags (<br />).
+        breaks: true, // Convert '\n' in paragraphs into <br>
+        langPrefix: 'lang-', // CSS language prefix for fenced blocks. Can be
```
[the remark this answers is not in the document]
Comment (alex-i-e) [reply src/shared/components/MarkDownEditor/MarkDownEditor.utils.ts]: done

File: src/shared/components/MarkDownEditor/MarkDownEditor.utils.ts [hunk 5]
```diff
@@ -78,177 +66,53 @@ MdEditor.config({
...
+      .use(katex)
+      .use(html5Embed, {
+        html5embed: {
+          useImageSyntax: true,
+        },
+      })
+      .use(markdownItImSize);
+  },
+  markdownItPlugins(plugins) {
+    return plugins.map((p) => {
```
Comment (IhorNesterenko99) [inline src/shared/components/MarkDownEditor/MarkDownEditor.utils.ts:104]:
```suggestion
    return plugins.map((plugin) => {
```

File: src/shared/components/MarkDownEditor/MarkDownEditor.utils.ts [hunk 6]
```diff
@@ -78,177 +66,53 @@ MdEditor.config({
...
+      if (p.type === 'image') {
+        return {
+          ...p,
+          options: {
+            ...p.options,
+            classes: 'my-class',
```
Comment (IhorNesterenko99) [inline src/shared/components/MarkDownEditor/MarkDownEditor.utils.ts:110]: maybe use more special class name (as I understood here is adding class to the image)?

**Review by IhorNesterenko99 — APPROVED** [review]

**Pull request merged**